snapcraft/internal/build_providers/_multipass/_instance_info.py

```diff
@@ -0,0 +1,81 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2018 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+from typing import TypeVar, Type
+
+from snapcraft.internal.build_providers import errors
+
+T = TypeVar('T', bound='InstanceInfo')
```

Comment:
Is there a need for this? You can just use `'InstanceInfo'` (the string) as the type; as long as it's valid after this file is processed, mypy will be happy with it.